Basic-Car-Maintenance/Shared/Models/AlertItem.swift

```diff
@@ -0,0 +1,45 @@
+//
+//  AlertItem.swift
+//  Basic-Car-Maintenance
+//
+//  Created by Omar Hegazy on 13/10/2023.
+//
+
+import Foundation
+import FirebaseFirestoreSwift
+import SwiftData
+
+struct AlertItem: Codable, Identifiable {
+    @DocumentID var id: String?
+    var actionTxt: String
+    var actionURL: String
+    var icon: String
+    var isOn: Bool
+    var message: String
+    var title: String
+    
+    enum CodingKeys: String, CodingKey {
+        case id
+        case actionTxt = "actionTex"
```

Comment:
```suggestion
        case actionTxt = "actionText"
```

typo, on my end, haha spell it out correctly, and I'll fix it in Firestore